### PR TITLE
VPA uses v1 API version

### PR DESCRIPTION
**Pull request by bskiba**
Also switches e2e tests to deploy the v1 crd.

### CODE REVIEW

File: vertical-pod-autoscaler/pkg/target/v1beta1_fetcher.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/*
```
Comment (jbartosik) [inline vertical-pod-autoscaler/pkg/target/v1beta1_fetcher.go]: This file looks unused - why is it part of the PR?

Comment (bskiba) [reply vertical-pod-autoscaler/pkg/target/v1beta1_fetcher.go]: Ah, thanks! Rebase confusion. Removed.

**Comment by bskiba** [comment]
Gah, I broke something while fixing the other thing, one more round.

**Comment by jbartosik** [comment]
Looks like some tests still use objects from wrong API version. Otherwise LGTM

**Pull request merged**